### ISSUE TITLE
Undeclared dependency Tie::IxHash

### ISSUE THREAD

**Issue by eserte** [perlpunk/YAML-PP-p5#6]
The test suite fails without Tie::IxHash:
```
...
#   Failed test 'YAML/PP/Schema/Tie/IxHash.pm loaded ok'
#   at t/00.compile.t line 66.
#          got: '512'
#     expected: '0'
Can't locate Tie/IxHash.pm in @INC (you may need to install the Tie::IxHash module) (@INC contains: /home/cpansand/.cpan/build/2019030321/YAML-PP-0.010_001-W9_LJt/blib/arch /home/cpansand/.cpan/build/2019030321/YAML-PP-0.010_001-W9_LJt/blib/lib /home/cpansand/.cpan/build/2019030321/YAML-PP-0.010_001-W9_LJt/blib/lib /home/cpansand/.cpan/build/2019030321/YAML-PP-0.010_001-W9_LJt/blib/arch /usr/perl5.22.4p/lib/site_perl/5.22.4/amd64-freebsd /usr/perl5.22.4p/lib/site_perl/5.22.4 /usr/perl5.22.4p/lib/5.22.4/amd64-freebsd /usr/perl5.22.4p/lib/5.22.4 .) at /home/cpansand/.cpan/build/2019030321/YAML-PP-0.010_001-W9_LJt/blib/lib/YAML/PP/Schema/Tie/IxHash.pm line 10.
BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2019030321/YAML-PP-0.010_001-W9_LJt/blib/lib/YAML/PP/Schema/Tie/IxHash.pm line 10.
Compilation failed in require at -e line 1.
# Looks like you failed 1 test of 23.
t/00.compile.t ............. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/23 subtests 
...
Can't locate Tie/IxHash.pm in @INC (you may need to install the Tie::IxHash module) (@INC contains: /home/cpansand/.cpan/build/2019030321/YAML-PP-0.010_001-W9_LJt/blib/lib /home/cpansand/.cpan/build/2019030321/YAML-PP-0.010_001-W9_LJt/blib/arch /usr/perl5.22.4p/lib/site_perl/5.22.4/amd64-freebsd /usr/perl5.22.4p/lib/site_perl/5.22.4 /usr/perl5.22.4p/lib/5.22.4/amd64-freebsd /usr/perl5.22.4p/lib/5.22.4 .) at /home/cpansand/.cpan/build/2019030321/YAML-PP-0.010_001-W9_LJt/blib/lib/YAML/PP/Schema/Tie/IxHash.pm line 10.
BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2019030321/YAML-PP-0.010_001-W9_LJt/blib/lib/YAML/PP/Schema/Tie/IxHash.pm line 10.
Compilation failed in require at t/38.schema-ixhash.t line 9.
BEGIN failed--compilation aborted at t/38.schema-ixhash.t line 9.
t/38.schema-ixhash.t ....... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
...
```

**Comment by perlpunk** [comment]
yep, thanks.
My plan was actually to make it optional but I forgot.
Fixed in master...

**Comment by perlpunk** [comment]
Released 0.010_002
